release_notes/ocp_3_10_release_notes.adoc

```diff
@@ -370,6 +370,14 @@ $ oc adm prune groups --sync-config=path/to/sync/config [<options>]
 ----
 See xref:../admin_guide/pruning_resources.adoc#pruning-groups[Pruning groups] for more information.
 
+[[ocp-310-local-ephemeral-storage]]
```

Comment:
@tmorriso-rh I added some draft text for this feature, the original PR mentioned adding it to the release notes. PTAL. Thanks!

Reply:
@bmcelvee This is the note we have in the ephemeral storage docs:

"This topic applies only if the ephemeral storage technology preview is enabled in {product-title} 3.10. This feature is disabled by default. If enabled, the {product-title} cluster uses ephemeral storage to store information that does not need to persist after the cluster is destroyed.  See Configuring Ephemeral Storage for more information."

Maybe include the part about the cluster using ephemeral storage to store information that does not need to persist after the cluster is destroyed? Just for consistency across docs. Thanks!

Reply:
Thanks @tmorriso-rh! I added that bit. :stars: